Makefile

```diff
@@ -74,6 +77,22 @@ down: ## remove stack (warning: it removes the database container)
 	@$(COMPOSE) down || echo WARNING: unable to remove the stack. Try to stop linked containers or networks first.
 .PHONY: down
 
+fixtures: 
+	@$(COMPOSE) up -d elasticsearch
+	@echo "Wait for database to be up..."
+	@$(WAIT_ES)
+	zcat ./fixtures/elasticsearch/lrs.json.gz | \
+	  python3 ./scripts/elasticsearch.py | \
```

Comment:
Use a `python` service from the `docker-compose.yml` to avoid having a project dependency.

Reply:
```suggestion
	  python3 ./scripts/patch_statements_date.py | \
```